app/containers/App/App.jsx

```diff
@@ -98,19 +99,40 @@ const App = ({
     handleUpgrade()
   }, [])
 
-  useEffect(
-    () => {
-      ipc.on('link', (event, url) => {
-        const { uri } = parseQuery(decodeURI(url))
-        if (uri) {
+  useEffect(() => {
+    ipc.on('link', (event, url) => {
+      const { uri } = parseQuery(decodeURI(url))
+      if (uri) {
+        if (store.getState()?.spunky?.auth?.data?.address) {
```

Comment:
We check to see if there is a loaded address in the state tree, if there is we immediately perform the route change otherwise render the notification and queue the route